src/lib/CMakeLists.txt

```diff
@@ -87,6 +87,8 @@ set(
     operators/table_wrapper.hpp
     operators/union_all.cpp
     operators/union_all.hpp
+    operators/set_union.cpp
```

Comment:
Not happy with SetUnion either. A set cannot hold duplicates, so this sounds more like "SELECT DISTINCT UNION SELECT DISTINCT"

Reply:
The Operator is essentially a wrapper around the same functionality as it is found in `std::set_union()` - and if the C++ guys settled on that name, maybe we should, too?

Reply:
I think "set" has a different meaning in their world. In the database world, it means "rows with distinct values" (https://courses.cs.washington.edu/courses/cse444/10sp/lectures/lecture16.pdf)